packages/react/src/components/Dialog/Dialog.tsx

```diff
@@ -275,5 +275,7 @@ Dialog.slotClassNames = {
 
 /**
  * A Dialog indicates a possible user action.
```

Comment:
I don't think that's good description of dialog.  We can look at other frameworks for better description, but perhaps something like, a dialog is used to interrupt the standard flow to get some particular input from the user.

Reply:
it is not - I have missed this one :)